frontend/src/lib/components/LemonModal/LemonModal.tsx

```diff
@@ -94,7 +96,7 @@ export function LemonModal({
         </>
     )
     return inline ? (
-        <div className="LemonModal ReactModal__Content--after-open">{modalContent}</div>
+        <div className={clsx('LemonModal ReactModal__Content--after-open', className)}>{modalContent}</div>
```

Comment:
I think this component should not have classNames as a prop otherwise in 6 months time we will once again be wrestling with custom overrides everywhere that could likely were not necessary in the first place. I think I commented above that this component could be wrapped by another that takes care of the positioning only

Reply:
That's a good point, removed this.